.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Primer Brand",
-    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:16",
+    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:18",
```

Comment:
Thanks @stamat, can we bump this to Node 20 at a minimum given 22 is now LTS?

Reply:
```suggestion
    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:20",
```

Reply:
@rezrah I had it first at 20 lol, but I was like maybe it's too much of an upgrade :D Yeah I agree legend, let's go with 20!